megazords/ios/MozillaAppServicesTests/LoginsTests.swift

```diff
@@ -6,36 +6,60 @@ import XCTest
 @testable import MozillaAppServices
 
 class LoginsTests: XCTestCase {
-    func getTestStorage() -> LoginsStorage {
+    var storage: LoginsStorage!
+    var encryptionKey: String!
+    var salt: String!
+
+    //This tests setup mimics closer to what fxiOS is doing
```

Comment:
nit: "closer" is good context for reviewing the PR, but in a few months time someone reading this is just going to think "closer than what?"; something like "as closely as we can" or similar might be better for a long-lived comment.